__tests__/app.test.js

```diff
@@ -45,25 +37,102 @@ describe('/api', () => {
+            expect(article).toMatchObject({
+                created_at: expect.any(String)
+            })
+        })
+    });    
+});
+
+describe('GET /api/articles', () => {
+    test('GET: 200 returns an array of all objects within articles without body property, sorted in descending order by creation date', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const {articles} = response.body
+            expect(articles.length).toBe(13)
+            articles.forEach(article => {
+                expect(article).toMatchObject({
                     title: expect.any(String),
                     topic: expect.any(String),
                     author: expect.any(String),
-                    body: expect.any(String),
                     created_at: expect.any(String),
                     votes: expect.any(Number),
-                    article_img_url:
-                    expect.any(String),
+                    article_img_url: expect.any(String),
+                    comments_count: expect.any(String)
+                })
+                expect(articles).toBeSorted({ descending: true });
+            })
+        })
+    });
+});
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('GET: 200 returns all comments associated with a specific article_id in ascending order based on creation date', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            const {comments} = response.body
+            comments.forEach(comment => {
+                expect(comment).toMatchObject({
+                    comment_id : expect.any(Number),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                    author: expect.any(String),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String)
+                    })
                 })
+                expect(comments).toBeSorted({ descending: false });
+            })
+    })
+})
+
+describe('Error Handling', () => {
+    describe('404: Bad Path', () => {
+        test('GET:404 sends an appropriate status and error message when given an invalid url', () => {
+        return request(app)
+        .get('/api/articules')
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe('Invalid URL');
             })
+        });
+        test('GET:404 sends an appropriate status and error message when given an invalid url', () => {
+            return request(app)
+            .get('/api/topices')
+            .expect(404)
+            .then((response) => {
+            expect(response.body.msg).toBe('Invalid URL');
+            });
+        });
+        test('GET:404 sends an appropriate status and error message when provided with a non-existent but valid id', () => {
```

Comment:
Errors related to an endpoint should be in the relevant describe block so that they are easy to find an clear that you've handled them.